### PR TITLE
Grammar edits_21_microbiome_community.qmd

### PR DESCRIPTION
**Pull request by jkc9886**
Grammar, spelling and punctuation edits

### CODE REVIEW

**Review by antagomir — APPROVED** [review]

**Comment by antagomir** [comment]
nice!

**Comment by jkc9886** [comment]
This PR is ready to be merged.

**Review by antagomir — APPROVED** [review]

**Comment by antagomir** [comment]
This seems OK - to merge, @TuomasBorman ?

File: inst/pages/21_microbiome_community.qmd [hunk 1]
```diff
@@ -199,7 +199,7 @@ sample_data <- data.frame(clusters = sample_clusters)
 # Order data so that it's same as in phylo tree
 sample_data <- sample_data[samples_ordered, , drop = FALSE] 
 
-# Order data based on 
+# Order data based on the phlyo tree
```
Comment (TuomasBorman) [inline inst/pages/21_microbiome_community.qmd]: typo

File: inst/pages/21_microbiome_community.qmd [hunk 2]
```diff
@@ -43,17 +43,17 @@ plotAbundance(tse_phylum, assay.type="relabundance", rank = "Phylum",
...
-horizontal axis represents samples and the vertical axis the
-taxa. The color of each intersection point represents abundance of a taxon
+Community composition can be visualized with a heatmap, where the
+horizontal axis represents samples and the vertical axis represents
+taxa. The color of each intersection point represents the abundance of a taxon
 in a specific sample.
 
 Here,  abundances are  first CLR  (centered log-ratio)  transformed to
-remove  compositionality bias. Then  Z  transformation  is applied  to
+remove  compositionality bias. Then, Standardize(or  'Z') transformation  is applied  to
```
Comment (TuomasBorman) [inline inst/pages/21_microbiome_community.qmd]: Standardize(or  'Z') --> "Then, standardization is applied..."

**Review by TuomasBorman — CHANGES_REQUESTED** [review]

File: inst/pages/21_microbiome_community.qmd (hunk 1 above)
Comment (jkc9886) [reply inst/pages/21_microbiome_community.qmd]: Sorry, I did not understand you here, do you mean i should the remove the added part?

File: inst/pages/21_microbiome_community.qmd (hunk 2 above)
Comment (jkc9886) [reply inst/pages/21_microbiome_community.qmd]: Done :)

File: inst/pages/21_microbiome_community.qmd (hunk 1 above)
Comment (antagomir) [reply inst/pages/21_microbiome_community.qmd]: phlyo is typo

File: inst/pages/21_microbiome_community.qmd (hunk 2 above)
Comment (antagomir) [reply inst/pages/21_microbiome_community.qmd]: resolve conversations when they are done. This way we can keep track on what has been dealt with.

File: inst/pages/21_microbiome_community.qmd (hunk 1 above)
Comment (jkc9886) [reply inst/pages/21_microbiome_community.qmd]: Oh no sorry I was confused, fixed it.

**Review by TuomasBorman — APPROVED** [review]

**Pull request merged**